libs/Theseus/src/dirk/dirk.jl

```diff
@@ -105,15 +114,18 @@ function init(
     du = zero(u)
     res = zero(u)
     u_tmp = similar(u)
+    tmp = similar(u)
     stages = ntuple(_ -> similar(u), Val(N))
     t = first(ode.tspan)
     iter = 0
+    # TODO: Refactor to provide method that re-uses the cache here.
+    kc = KrylovConstructor(res)
```

Comment:
This comes from Krylov.jl?

Reply:
Yes, it's a good place. Could you please adapt the `using` lines like https://github.com/NumericalMathematics/Ariadne.jl/blob/d6fa3144d6dc8a63d2bd02db3d6af35b5b6bb4ed/libs/Theseus/src/Theseus.jl#L6 to explicitly list the things we want to use? This avoids possible breakage by having conflicting imports from new releases.